test/unit/app/utils/test_kenya_state_machine.py

```diff
@@ -199,6 +203,7 @@ def test_authorize_pin(test_client, init_database, session_factory, user_input,
     assert user.failed_pin_attempts == failed_pin_attempts
 
 
+#TODO: also test that users pin does not become the new thing after
```

Comment:
In the event that this TODO and the one above https://github.com/teamsempo/SempoBlockchain/pull/55/files#r347084848 refer to those tests, then I think the appropriate place for those tests to be is in the utils.user file where those methods are. So I'll put those tests there.